tests/integration/kubernetes/gha-run.sh

```diff
@@ -548,6 +548,10 @@ function cleanup_nydus_snapshotter() {
 	kubectl delete -f "misc/snapshotter/nydus-snapshotter-rbac.yaml"
 	kubectl get namespace nydus-system -o json | jq 'del(.spec.finalizers)' | kubectl replace --raw "/api/v1/namespaces/nydus-system/finalize" -f - || true
 	popd
+	for i in `sudo ctr -n k8s.io snapshot --snapshotter nydus list | grep -v KEY | cut -d' ' -f1 || true`; do
+		sudo ctr -n k8s.io snapshot --snapshotter nydus rm $i
+	done
+	sudo rm -rf "/var/lib/containerd-nydus" || true
```

Comment:
This should be done by the nydus-snapshotter.
Any specific reason to enforce it here?

In fact, please, take a look at: https://github.com/containerd/nydus-snapshotter/blob/ce4848e4439b79723b0635e994b5d9991aa88819/misc/snapshotter/snapshotter.sh#L195

So, isn't the snapshotter doing exactly the same thing on its own side?

Reply:
I think these guys ran into the `Error: failed to create containerd container: create instance 6: object with key "6" already exists: unknown` that you also saw on the TDX node. Did you end up doing anything to fix that besides local cleanup?

Reply:
Regardless of the error they hit, this should be fixed on the nydus snapshotter side, *not* on the Kata Containers side.

I didn't hit the issue on TDX, but that could happen there as well.  I hit the issue on a local VM when trying the snapshotter before the release.  That cleanup was enough to get me moving from the error.

Reply:
@wainersm pointed out in the slack thread a snippet to clean up the `content`, that's not relevant for the error that Tobin mentioned to be happening, but rather to remove the image from overlayfs snapshotter and allowing to pull the image with nydus, which is an approach **we do not want to take here**, as that's just papering over real issues.